kaggle_environments/envs/llm_20_questions/llm_20_questions.py

```diff
@@ -75,6 +75,66 @@ def answerer_agent(obs):
 
 agents = {"guesser": guesser_agent, "answerer": answerer_agent}
 
+def guesser_action(active, inactive, step):
+    guessed = False
+    if not active.action:
+        active.status = "ERROR"
+    elif active.observation.turnType == "ask":
+        active.observation.questions.append(active.action)
+        inactive.observation.questions.append(active.action)
+    elif active.observation.turnType == "guess":
+        active.observation.guesses.append(active.action)
+        inactive.observation.guesses.append(active.action)
+    if active.action and keyword_guessed(active.action):
+        guessed = True
+        score = 20 - int(step / 3)
+        active.reward = score
+        inactive.reward = score
+        active.status = "DONE"
+        inactive.status = "DONE"
+        active.observation.keyword = keyword
+        active.observation.category = category
+    inactive.observation.keyword = keyword
+    inactive.observation.category = category
+    return guessed
+
+def answerer_action(active, inactive):
+    active.observation.keyword = keyword
+    active.observation.category = category
+    response = active.action
+    if not response:
+        response = "none"
+        active.status = "ERROR"
+    elif response.lower().__contains__("yes"):
```

Comment:
nit: Assuming this is a string, could you just do `"yes" in response.lower()`?  More generally `__contains__` is a bit of a smell, but you could put a comment explaining why

Also nit that I think Google style is to use single quotes for strings, but that ship may have sailed a while ago :)